web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -45,7 +45,7 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
   } = useFormContextHelper(createProfileFieldErrorMap());
 
   useEffect(() => {
-    const current = updateQueue?.current();
+    const current = userDataFromHook.userData;
```

Comment:
Why can't we just use `userData` for the instances of this variable?

Reply:
We can for this one! Updating